doc/Core functionalities.md

```diff
@@ -1 +1,36 @@
-Hello
\ No newline at end of file
+# Wanted functionalities for Pithekos
+## Results of an initial thinking (12/2024)
+- Resources management
+    - Search
```

Comment:
Search what? I think search has to look different for local and remote, unless we completely merge the two which we can't do given that remote is expected to not exist most of the time.

Reply:
At that question, let's go with "unsure" for now. It's still possible to think about something close to Sunteleo, where the downloadable resources show up when online. A single research can simplify things. We'll have to look into that at some point.